Export Import Views/ExportImportViews.js

```diff
@@ -0,0 +1,158 @@
+tau
+    .mashups
+    .addDependency('jQuery')
+    .addDependency('Underscore')
+    .addDependency('tau/configurator')
+    .addMashup(function ($, _, configurator) {
+
+        'use strict';
+
+        $('head').append([
+            '<style>',
+            '.mashup-import-view .drop-zone {',
+            'border: 2px dashed #bbb;',
+            '-moz-border-radius: 5px;',
+            '-webkit-border-radius: 5px;',
+            'border-radius: 5px;',
+            'padding: 25px;',
+            'text-align: center;',
+            'color: #bbb;',
+            'margin-top: 24px;',
+            'margin-bottom: 12px;',
+            '}',
+            '.mashup-import-file {',
+            ' display:none;',
+            '}',
+
+            '</style>'
+        ].join('\n'));
+
+        var reg = configurator.getBusRegistry();
+        var bs;
+
+        var addBusListener = function (busName, eventName, listener) {
+
+            reg.on('create', function (e, data) {
+
+                var bus = data.bus;
+                if (bus.name === busName) {
+                    bus.on(eventName, listener);
+                }
+            });
+
+            reg.on('destroy', function (e, data) {
+
+                var bus = data.bus;
+                if (bus.name === busName) {
+                    bus.removeListener(eventName, listener);
+                }
+            });
+
+            reg.getByName(busName).done(function (bus) {
+                bus.on(eventName, listener);
+            });
+        };
+        var $exportButton;
+
+
+        var changeHandler = function () {
+            bs.boardSettings
+                .get([])
+                .then(prepareButton);
+        };
+
+
+        var prepareButton = function (res) {
+            var json = _.omit(res, 'createdAt', 'id', 'isShared', 'menuIsVisible', 'menuNumericPriority', 'ownerIds', 'acid', 'edit');
+            var blob = new Blob([JSON.stringify(json)], {type: "application/json"});
+            var url = URL.createObjectURL(blob);
+            if ($exportButton) {
+                $exportButton.prop({
+                    href: url,
+                    download: res.name + '.json'
+                })
+            }
+
+        };
+
+
+        addBusListener('board.toolbar', 'boardSettings.ready', function (e, boardSettings) {
+            bs = boardSettings;
+            bs.boardSettings
```

Comment:
boardSettings are somewhat deprecated IMO. There is `configurator.getViewApiService()`, which has `.createViewMenuItem(data)` method. With it you also won't need to subscribe to boardSettings changes.

"Export" button can also call its `getViewMenuItem(id)` method.